ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx

```diff
@@ -1,28 +1,59 @@
 import React from 'react';
 import { Card, CardBody, CardTitle, Flex, Grid, GridItem, Text } from '@patternfly/react-core';
-import { SVGIconProps } from '@patternfly/react-icons/dist/esm/createIcon';
+import { gql } from '@apollo/client';
 
 import SeverityIcons from 'Components/PatternFly/SeverityIcons';
 
 import { VulnerabilitySeverity } from 'types/cve.proto';
 import { vulnerabilitySeverityLabels } from 'messages/common';
 
-export type BySeveritySummaryCardProps = {
-    className?: string;
-    title: string;
-    severityCounts: Omit<Record<VulnerabilitySeverity, number>, 'UNKNOWN_VULNERABILITY_SEVERITY'>;
-    hiddenSeverities: Set<VulnerabilitySeverity>;
-};
-
 const severitiesCriticalToLow = [
     'CRITICAL_VULNERABILITY_SEVERITY',
     'IMPORTANT_VULNERABILITY_SEVERITY',
     'MODERATE_VULNERABILITY_SEVERITY',
     'LOW_VULNERABILITY_SEVERITY',
 ] as const;
 
+const severityToQuerySeverityKeys = {
+    CRITICAL_VULNERABILITY_SEVERITY: 'critical',
+    IMPORTANT_VULNERABILITY_SEVERITY: 'important',
+    MODERATE_VULNERABILITY_SEVERITY: 'moderate',
+    LOW_VULNERABILITY_SEVERITY: 'low',
+} as const;
+
 const fadedTextColor = 'var(--pf-global--Color--200)';
 
+export type ResourceCountsByCveSeverity = {
+    critical: { total: number };
+    important: { total: number };
+    moderate: { total: number };
+    low: { total: number };
+};
+
+export const resourceCountByCveSeverityFragment = gql`
```

Comment:
So although this is exported, it isn't actually used anywhere. All the components that use this summary card also use the "Cves by status" summary card, which depends on the following fragment:
```graphql
    fragment ResourceCountsByCVESeverityAndStatus on ResourceCountByCVESeverity {
        critical {
            total
            fixable
        }
        important {
            total
            fixable
        }
        moderate {
            total
            fixable
        }
        low {
            total
            fixable
        }
    }
```
.

Since the more verbose fragment is used anyway, we can just pass the resulting data to this component without issue.

There is no reason we couldn't explicitly use _both_ fragments in calling queries, like:
```typescript
const imageVulnerabilitiesQuery = gql`
    ${resourceCountByCveSeverityFragment}
    ${resourceCountByCveSeverityAndStatusFragment}
    query getImageCoreVulnerabilities($id: ID!, $query: String!, $pagination: Pagination!) {
        image(id: $id) {
            imageCVECountBySeverity(query: $query) {
                ...ResourceCountsByCVESeverity
                ...ResourceCountsByCVESeverityAndStatus
            }
        }
    }
`;
```
but it wouldn't do anything other than send a little bit of extra text to the server. The main reason for doing this is it wouldn't make `<BySeveritySummaryCard>` implicitly dependent on data for another card.

Any strong opinions on:
1. Explicitly including the extra fragment in the query as above, for completeness (maybe with a comment)
2. Not including the extra fragment in the query and deleting the unused exported fragment in this component
3. Do nothing, and leave it as-is

Reply:
hmm i'm torn between 2 & 3 -- i feel like if the fragment is not currently being used, we can leave it out unless there is a scenario where the `BySeveritySummaryCard` would not have access to that passed through data. i suppose for now i'm okay with 2 until we find a need to make this component more independent/context unaware

Reply:
2 it is 🪓